migration_scripts/to_version_7_1_12/index.js

```diff
@@ -0,0 +1,110 @@
+/*
+ *    Copyright (c) 2023, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ *    This software is licensed under the Apache License, Version 2.0 (the
+ *    "License") as published by the Apache Software Foundation.
+ *
+ *    You may not use this file except in compliance with the License. You may
+ *    obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *    License for the specific language governing permissions and limitations
+ *    under the License.
+ */
+
+const libphonenumber = require('libphonenumber-js/max');
+
+// Update the following credentials before running the script
+const DB_HOST = "";
+const DB_USER = "";
+const DB_PASSWORD = "";
+const DB_NAME = "";
+const CLIENT = ""; // Use "pg" for PostgreSQL and "mysql2" for MySQL DB
+
+if (!DB_HOST || !CLIENT) {
+  console.error('Please update the DB_HOST, DB_USER, DB_PASSWORD, DB_DATABASE and CLIENT variables before running the script.');
+  return;
+}
+
+const knex = require('knex')({
```

Comment:
whats the default connection and query timeout for this? We should keep them large (min 10 seconds for both)

Reply:
default `acquireConnectionTimeout` is `60000ms`

Reply:
There is no timeout for query but based on the suggestion I have also added a  `QUERY_TIMEOUT` of 60s.